src/app/actions.ts

```diff
@@ -0,0 +1,37 @@
+"use server";
```

Comment:
can we have all the code client side or static? adonis is the backend so having server actions have more code run on the server is a bit odd.

I recommend using ky to make fetch requests, syntax is much cleaner. more info: https://www.adios-axios.com/

Reply:
done